src/client/opamClient.ml

```diff
@@ -752,6 +752,62 @@ let init_checks ?(hard_fail_exn=true) init_config =
   if hard_fail && hard_fail_exn then OpamStd.Sys.exit_because `Configuration_error
   else not (soft_fail || hard_fail)
 
+(* Check sandboxing script call. If it errors or unattended output, disable
```

Comment:
Not sure I would have chosen this module for this, since it's at the moment concerned with package-handling stuff and the switch state... but I don't have a much better proposition ?

Reply:
moved to `OpamAuxCommand`